packages/cloud_firestore/cloud_firestore_platform_interface/lib/src/method_channel/method_channel_query.dart

```diff
@@ -212,6 +213,13 @@ class MethodChannelQuery extends QueryPlatform {
     });
   }
 
+  @override
+  AggregateQueryPlatform count() {
+    return MethodChannelAggregateQuery(
+      this,
```

Comment:
do we have a styleguide for this?

Reply:
We don't AFAIK, it just follows the same implementation style as the rest of the plugins.